src/applications/vaos/tests/reducers/appointments.unit.spec.js

```diff
@@ -86,13 +95,14 @@ describe('VAOS reducer: appointments', () => {
           },
         ],
         [
+          // appointment times scheduled 395 days into the future should show
           {
-            appointmentTime: '05/29/2099 05:30:00',
+            appointmentTime: '05/29/2021 05:30:00',
```

Comment:
Why did you set this one to a specific date, rather than adding 394 days to the current date like above? It looks like before this was set to a far future date that would be excluded by the filter.

Reply:
Hi @jbalboni, good point. I'll change the fixed date to use 394+ days to the current date. Also add another appointment data far into the future to be excluded by filter.